src/lib/libraries/sprites.json

```diff
@@ -11784,28 +12182,103 @@
             "currentCostume": 0,
             "costumes": [
                 {
-                    "assetId": "89aa5332042d7bbf8368293a4efeafa4",
-                    "name": "magicwand",
+                    "assetId": "90fa2ad340edc6e6ba963710feef940e",
+                    "name": "Luca-a",
                     "bitmapResolution": 1,
-                    "md5ext": "89aa5332042d7bbf8368293a4efeafa4.svg",
+                    "md5ext": "90fa2ad340edc6e6ba963710feef940e.svg",
                     "dataFormat": "svg",
-                    "rotationCenterX": 41,
-                    "rotationCenterY": 18
+                    "rotationCenterX": 44.518938859952925,
+                    "rotationCenterY": 130.28487970293187
+                },
+                {
+                    "assetId": "18dfad514602a4907502c7c84861b24e",
+                    "name": "Luca-b",
+                    "bitmapResolution": 1,
+                    "md5ext": "18dfad514602a4907502c7c84861b24e.svg",
+                    "dataFormat": "svg",
+                    "rotationCenterX": 41.801770753585856,
+                    "rotationCenterY": 130.2642790555675
+                },
+                {
+                    "assetId": "963cb82687acaf5de53a22b287192723",
+                    "name": "Luca-c",
+                    "bitmapResolution": 1,
+                    "md5ext": "963cb82687acaf5de53a22b287192723.svg",
+                    "dataFormat": "svg",
+                    "rotationCenterX": 50.565256229860694,
+                    "rotationCenterY": 130.02168173555543
                 }
             ],
             "sounds": [
                 {
                     "assetId": "83a9787d4cb6f3b7632b4ddfebf74367",
-                    "name": "pop",
+                    "name": "Pop",
```

Comment:
Noticed this weird difference between this "Pop" with capital P and the "pop" sound with lowercase p in the Magic Wand sprite...

Reply:
Right! we are a bit inconsistent with capitalization across the libraries, unfortunately. There are a couple of pre-existing instances of "Pop" but they are almost all "pop".